### ISSUE TITLE
"Lawsuit" not properly counting towards "Warmonger" award

### ISSUE THREAD

**Issue by Erostal** [terraforming-mars/terraforming-mars#6310]
Hello,

In a recent game it appears that the "Lawsuit" card isn't counting (or at least isn't displayed counting) towards the "Warmonger" award.

In the below linked game, the Red player plays Fish, Birds, Biomass Combustors, Deimos Down, and Lawsuit (Lawsuit played in Gen 7). The counter for the Warmonger award shows 4, while I believe it should be showing 5. We couldn't verify if it actually counted or not as the 2nd place in the award was at 3 cards.

This may have something to do with how Lawsuit is played - it going into another player's played event stack?

Thanks for all your work on this tool!

Game ID: g420be0779238
Game Log attached
[Terraforming Game Log.txt](https://github.com/terraforming-mars/terraforming-mars/files/13794759/Terraforming.Game.Log.txt)

**Comment by Erostal** [comment]
I don't believe this is correct for the Warmonger award. The description reads "Play the most cards that reduce other players' resources or production", not something along the lines of "Have the most cards **in play** that reduce...". Comparing this to the description of Celebrity which reads "Have the most project cards in play with a base of at least 20 ME (not events)." which specifically excludes events as well as specifying the "in play".

Also if it's not counting events there is still an issue, as the only 2 cards the red player had in play that reduce others were Birds and Fish, which would total up to 2, not the shown 4. Similarly, the green player has a count of 3, but only 1 blue or green card that reduces (Asteroid Mining Consortium), but including events does have 3.

**Comment by kberg** [comment]
Thanks. I've taken some time to review this, and events don't count toward awards. So I'll not include Lawsuit, and I'll also remove the other events.

**Comment by naanlizard** [comment]
Why would events not count? It says "Play the most cards", not "play the most non-event cards"

**Comment by kberg** [comment]
There's a rule in the rulebook that makes it clear: events don't count once
they are played, unless specifically mentioned. I think there are one or
two places where something explicit says that events are not included, but
that's just repeating the rule, and it doesn't mean the rule needs to be
repeated everywhere else.

**Comment by Erostal** [comment]
Hi - I saw that in the April Changelog Warmonger was flagged to include events (which is great). In my game today, I believe I encountered three issues with the displayed count under the award (e.g. where it shows where each player ranks for the award):

- It didn't seem to count the card "Public Sponsored Grant". This may be intentional as PSG removes 2M€ from all players, but based on the wording of the Warmonger description, it should fit this case. It did seem to be counting another player's "Dust Storm", which removes all players energy resources, so I was assuming PSG would work similarly
- I played the "Asteroid" card, then returned it to hand with the "Astra Mechanica" card. I didn't end up playing it again, but it was removed from the Warmonger count once it was back in my hand. The Warmonger wording is "Play the most cards...", not have in play, so it should have still counted, or the wording should be updated to be made more clear. If I did end up playing "Asteroid" again, I'm unsure if it should count twice - I would lean towards yes that it should, but I see the case for it just counting once
- An opponent played "Law Suit" against me and it didn't appear to count towards his Warmonger score

We didn't end up funding Warmonger (partially due to this issue), and I unsure if you're able to go back and see the Award display counts at checkpoints/generations before the end, but I've put the game details below.

Link to the game: https://terraforming-mars.herokuapp.com/the-end?id=p33fadb1d3330
Game Id: g2fdc82aafa65
Public Sponsored Grant and Astra Mechanica were both played by the "Rob" player in the game, and Lawsuit was played by the "Corbin" player but ended in the stack of "Rob"

If you want me to pop this into a new ticket, happy to do so. As always, thanks for all the hard work in keeping this site up and updated!

**Comment by kberg** [comment]
Ever since dealing with milestones like Thawer which requires players to keep track by marking tokens, I think it's right to go ahead and track Law Suit as a valid Warmonger card, by the player themselves.

**Comment by rusliksu** [comment]
Thanks for fixing the Law Suit interaction in #8046. I understand the Thawer precedent, but the current implementation now mixes two counting models.
Law Suit is history-based: the player who played it keeps the Warmonger credit after the card leaves play, and replaying the same physical card can add another credit without removing the first one. Ordinary eligible cards are tableau-based and stop counting when they leave play through Astra Mechanica, Patent Manipulation, Project Workshop, Odyssey, or Playwrights.
Could Warmonger use one consistent rule for all eligible cards—either score the final tableau and change the wording to “Have the most cards that reduce other players’ resources or production, including events”, or keep “Play the most...” and persistently track every eligible card when it is played?
I lean toward final tableau state because it matches how awards are normally scored and avoids additional bookkeeping. Once a direction is chosen, I can prepare a PR.
